devproject/settings.py

```diff
@@ -415,6 +416,7 @@
 
 EMAIL_BACKEND = "django.core.mail.backends.console.EmailBackend"
 
+CELERY_BROKER_URL = ""
```

Comment:
This should be `redis://redis:6379/0`. `redis://` part tells what broker to use, `redis:6379` means "on host `redis` and port `6379`". By default docker-compose services host is named after service itself, and `6379` is default port used by Redis. `/0` means "first database in Redis".